kaldb/src/main/java/com/slack/kaldb/metadata/snapshot/SnapshotMetadataStore.java

```diff
@@ -6,7 +6,7 @@
 import org.slf4j.LoggerFactory;
 
 public class SnapshotMetadataStore extends PersistentMutableMetadataStore<SnapshotMetadata> {
-  public static final String SNAPSHOT_METADATA_STORE_ZK_PATH = "/snapshots";
+  public static final String SNAPSHOT_METADATA_STORE_ZK_PATH = "/snapshot";
```

Comment:
This is the only ZK path that is plural, the rest are singular (search, replica, cacheSlot, etc)

Reply:
+1 for consistency.